.circleci/config.yml

```diff
@@ -90,6 +90,10 @@ jobs:
       NAMEDOTCOM_URL: api.name.com
       NAMEDOTCOM_USER: dnscontroltest
 
+      TRANSIP_DOMAIN: $TRANSIP_DOMAIN
```

Comment:
Any `X: $X` statement can be eliminated (I need to update the docs to mention this).  In your case, that eliminates all of these.

Reply:
Fixed. 👌